src/ResourceManager/DataFactories/Commands.DataFactoryV2.Test/ScenarioTests/IntegrationRuntimeTests.ps1

```diff
@@ -113,6 +113,7 @@ function Test-SsisAzure-IntegrationRuntime
             -CatalogPricingTier 'S1' `
             -MaxParallelExecutionsPerNode 1 `
             -LicenseType LicenseIncluded `
+            -Edition Enterprise `
```

Comment:
Please add the SetupScriptContainerSasUri parameter as well, and add an assert to ensure they are set properly.

Reply:
I add SetupScriptContainerSasUri , but did add assert as this case is not enabled now, per discussion with ADF team, they will help to enable the case, there is a task to track this, and it is same case for License Type

Reply:
If the case is not enabled now, why is the user allowed to add this parameter?  I would think you don't want to merge this change until the case is enabled.

Reply:
No, I mean test case is not enabled, the feature is enabled,but the test case is not enabled yet.
Refer to this issue for detail, https://github.com/Azure/azure-powershell/issues/5327

Can you let me know if you have any more comments?

Reply:
This should be good to merge once the nuget package is published.